doc/uftrace-report.md

```diff
@@ -23,45 +23,45 @@ OPTIONS
 :   Report thread summary information rather than function statistics.
 
 -s *KEYS*[,*KEYS*,...], \--sort=*KEYS*[,*KEYS*,...]
-:   Sort functions by given KEYS.  Multiple KEYS can be given, separated by comma (,).  Possible keys are 'total' (time), 'self' (time), 'call', 'avg', 'min', 'max'.  Note that first 3 keys should be used when neither of '--avg-total' nor '--avg-self' is used.  Likewise, the last 3 keys should be used when either of those option is used.
+:   Sort functions by given KEYS.  Multiple KEYS can be given, separated by comma (,).  Possible keys are `total` (time), `self` (time), `call`, `avg`, `min`, `max`.  Note that the first 3 keys should be used when neither of `--avg-total` nor `--avg-self` is used.  Likewise, the last 3 keys should be used when either of those options is used.
 
 \--avg-total
-:   Show average, min, max of each functions total time.
+:   Show average, min, max of each function's total time.
 
 \--avg-self
-:   Show average, min, max of each functions self time.
+:   Show average, min, max of each function's self time.
 
 \--diff=*DATA*
-:   Report difference between the input trace data and the given DATA.
+:   Report differences between the input trace data and the given DATA.
 
 \--sort-column=*IDX*
-:   When --diff option is used, 3 columns will be shown for each total time, self time and call count.  This is option is to select the index of column to be used as a sort key.  The index 0 is for original data given by --file option, and index 1 is for data given by --diff option, and index 2 is for (percentage of) difference between the two data.
+:   When `--diff` is used, 3 columns will be shown: total time, self time and call count.  This option selects the index of the column to be used as a sort key.  Index 0 is for original data given by the `--file` option, index 1 is for data given by the `--diff` option, and index 2 is for (percentage) differences between the two data.
 
 -k, \--kernel
-:   Trace kernel functions as well as user functions.  Only kernel functions inside user functions will be shown.
+:   Show kernel functions as well as user functions.  Only kernel functions called inside user functions will be shown.
 
 --kernel-full
-:   Show all kernel functions called outside of user functions.  Implies \--kernel option.
+:   Show all kernel functions, including those called outside of user functions.
```

Comment:
I did that just because the `--kernel` description specifically says "Only kernel functions called inside user functions will be shown". If `--kernel-full` implies `--kernel`, then we would expect both "Show all kernel functions, including those called outside of user functions" and "Only kernel functions called inside user functions will be shown", which is confusing.